components/link/link.js

```diff
@@ -1,8 +1,11 @@
 import '../colors/colors.js';
-import { css, html, LitElement } from 'lit';
+import { css, html, LitElement, unsafeCSS } from 'lit';
 import { classMap } from 'lit/directives/class-map.js';
 import { FocusMixin } from '../../mixins/focus-mixin.js';
 import { ifDefined } from 'lit/directives/if-defined.js';
+import { isFocusVisibleSupported } from '../../helpers/focus.js';
+
+const focusPseudoClass = isFocusVisibleSupported() ? 'focus-visible' : 'focus';
```

Comment:
Rather than duplicating the `'focus-visible' : 'focus'` logic in multiple files, what are your thoughts on something like:

```js
import { getFocusPseudoClass } from '../../helpers/focus.js';

// This could probably be embedded directly wherever it's used, but might be
// a small perf benefit to do it once at the top instead of e.g. in a render function...
const focusPseudoClass = getFocusPseudoClass();
```

Reply:
Yep. That's valid. I can't think of any reason why we'd need to do it in each place that needs it.

Reply:
I've updated with your suggestion. Verified that the `isFocusVisibleSupported` only gets called once per page, and `getFocusPseudoClass` only gets called once per module, no matter how many instances of the component there are.